src/ralph/security/views.py

```diff
@@ -16,7 +17,14 @@
 logger = logging.getLogger(__name__)
 
 
+@lru_cache()
```

Comment:
this cache will not work right now - the param to this function has to be type, not object